.github/release-template.md

```
@@ -26,60 +26,60 @@
 For support on installation and help please visit the following web pages:
 
-[RaspberryMatic - Documentation :us:](https://github.com/jens-maus/RaspberryMatic/wiki/en.Home)
-[RaspberryMatic - Discussions :us:](https://github.com/jens-maus/RaspberryMatic/discussions)
-[RaspberryMatic - Dokumentation :de:](https://github.com/jens-maus/RaspberryMatic/wiki)
-[RaspberryMatic - Forum :de:](https://homematic-forum.de/forum/viewforum.php?f=65)
+[OpenCCU - Documentation :us:](https://github.com/OpenCCU/OpenCCU/wiki/en.Home)
+[OpenCCU - Discussions :us:](https://github.com/OpenCCU/OpenCCU/discussions)
+[OpenCCU - Dokumentation :de:](https://github.com/OpenCCU/OpenCCU/wiki)
+[OpenCCU - Forum :de:](https://homematic-forum.de/forum/viewforum.php?f=65)
 
 ## 📦 Download:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)